content/scout/guides/vex.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Suppress image vulnerabilities with VEX
+description: An introduction to using VEX with Docker Scout
+keywords: scout, vex, Vulnerability Exploitability eXchange, openvex, vulnerabilities, cves
+---
+
+In this guide, you will learn about:
+
+- How VEX can help you suppress non-applicable or fixed vulnerabilities found in your images
+- How to create VEX documents and statements
+- How to apply and consume VEX data with Docker Scout
+
+## Prerequisites
+
+If you want to follow along the steps of this guide, you'll need the following:
+
+- Docker Desktop version 4.27.2 or later
+- Git
+- A Docker account
+- A GitHub account
+
+## Introduction to VEX
+
+Just because a software product contains a vulnerable component
+doesn't mean that the vulnerability is exploitable.
+For example, the vulnerable component might never be
+loaded into memory when the application runs.
+Or the vulnerability might have been fixed,
+but the system that detects the vulnerability is unaware.
+
+Vulnerability Exploitability eXchange (VEX) is a way to add context
```

Comment:
Suggest we move this statement to the intro section for those users who are new to VEX.